### DISCUSSION TITLE
RP.Math.Vector3 Build Dispatcher Exception when Build

### DISCUSSION THREAD

**Discussion by frickyto** [terjeio/ioSender#387] [Q&A]
Hello, I trying build the version of IoSender 2.0.44, I correct some problems firstly over Themes but when I resolve this and install all dependencies, the app fail at start and show a textbox saying "Unhandled exception occured: can´t load the file or build ´RP.Mathe.Vector3, Version=1.3.0.0,....... The system can´t find the specified file´ Then Accept and the app close.
I check if the dependence (.DLL File) is in the proyect, and it´s ok but fail. Can help me please?

UPDATE:
I check about some LIBS don´t copy to the compile directory and fail, for example, RP.Math.Vector3.dll and RP.Math.dll, if I manual copy from the pre-build app, It´s works.

**Comment by terjeio** [comment]
> if I manual copy from the pre-build app, It´s works.

Copy Local in Reference Properties for the dll is set to False?

![image](https://github.com/terjeio/ioSender/assets/20260062/649c6bf7-1afd-4121-989d-b33bb37c03be)

**Reply by frickyto** [reply]
I check and in one of the proyect was True but in other, that build after was False, and when build don´t copy de dll, I correct all references of this DLL to True and now works perfect. Thanks!!!!
Sorry for my English, I speak Spanish, I´m from Argentina.
Regards!!!
